webapp/components/test-file-results.js

```diff
@@ -270,8 +270,13 @@ class TestFileResults extends WPTFlags(LoadingState(PathInfo(
 
   resultsURL(testRun, path) {
     path = this.encodeTestPath(path);
-    // This is relying on the assumption that result files end with '-summary.json.gz'.
-    const resultsBase = testRun.results_url.slice(0, testRun.results_url.lastIndexOf('-summary.json.gz'));
+    // This is relying on the assumption that result
+    // files end with '-summary.json.gz' or '-summary_v2.json.gz'.
+    let resultsSuffix = '-summary.json.gz';
+    if (!testRun.results_url.contains('-summary')) {
```

Comment:
Question about this part - if the `results_url` contains `-summary`, it is guaranteed to be the `v2` format? Is this a unique identifier for the `v2` format? if not, do we have existing files with the `-summary.json.gz` format and do we  have a backward compatibility?

Reply:
Ah, this is a logic error on my part I forgot to replace, and this statement needs to check if the `results_url` contains `resultsSuffix`. Sorry fort the confusion!

The logic is supposed to be:
- Check for `summary.json.gz` substring in the url (signifies old format). Use this to find the index if it exists.
- If it does not exist in the url, it must be the new format. Instead use `summary_v2.json.gz`.